src/main/java/org/example/persistence/sql/SQLDialect.java

```diff
@@ -41,4 +41,17 @@ public static String getTableNameForSelect(Class<?> clss) {
         String tableName = AnnotationUtils.getTableName(clss);
         return String.format("SELECT * FROM %s WHERE id=?", tableName);
     }
+    public static String getTableAndColumnNamesForUpdate(Class<?> cls){
+        Field[] declaredFields = cls.getDeclaredFields();
+        List<String> columnNames = new ArrayList<>();
+        List<String> parameters = new ArrayList<>();
+        for(Field declaredField: declaredFields) {
+            if (!declaredField.isAnnotationPresent(Id.class)) {
+                columnNames.add(AnnotationUtils.getColumnName(declaredField));
+                parameters.add("?");
+            }
+        }
```

Comment:
Just a suggestion for the duplicate code but please feel free to do it whatever fits you the best:

lines 44-53 - make this method private and name it only getTableAndColumnNames;

add these lines:         List< String > tableColumnsAndParameters = new ArrayList<>();
        tableColumnsAndParameters .add(AnnotationUtils.getTableName(clss));
        tableColumnsAndParameters .add(String.join(", ", columnNames));
        tableColumnsAndParameters .add(String.join(", ", parameters));

and make it return list: tableColumnsAndParameters;

then this method can be easily used in methods like sqlInsert(Class< ? > clss) and sqlUpdate(Class< ? > clss) with bodies with only two lines:
List< String > someName = getTableAndColumnNames(clss);
return String.format("INSERT INTO %s (%s) values(%s)", someName.get(0), someName.get(1), someName.get(2));

and almost the same for the update:
-//-
return String.format("UPDATE %s SET %s = (%s)  WHERE id = ?", someName.get(0), someName.get(1), someName.get(2));